CHANGELOG.md

```diff
@@ -3,11 +3,17 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.4.0dev
+## v1.4.0 - 2023-04-01
```

Comment:
When do you expect to release?
Today or April the first?

Reply:
Indeed. Let's plan it for next Monday, hopefully ;)